doc/commands/download.8.rst

```diff
@@ -45,7 +45,14 @@ Options
     | To be used together with ``--resolve``, it downloads all dependencies, not skipping the already installed ones.
 
 ``--destdir=<path>``
-    Set directory used for downloading packages to. Default location is to the current working directory.
+    | Set directory used for downloading packages to. Default location is to the current working directory.
+
+``--url``
+    | Prints the list of urls where the rpms can be downloaded instead of downloading.
+
+``--urlprotocol``
+    | To be used together with ``--url``, it filters out the urls to the specified url protocols: {http, https, ftp, file}
```

Comment:
The option only accepts a single value. On the other hand, the option can be used multiple times. Could you please rewrite this documentation to make it clear. Also please spell URL with capitals. E.g.: 

~~~~
To be used together with ``--url``. It filters out the URLs to the specified protocol: ``http``, ``https``,
``ftp``, or ``file``. This option can be used multiple times.
~~~~